### DISCUSSION TITLE
Pass system property to native app

### DISCUSSION THREAD

**Discussion by amarkevich** [infinispan/infinispan#11217] [Q&A] [answered]
infinispan-quarkus-server-runner 14.0.13.Final native (powered by Quarkus 2.13.7.Final)

According to suggestion

$ ./bin/server.sh --help
-D<name>=<value>              Sets a system property to the specified value.

$ ./bin/server.sh -Dkey=value
ISPN090002: Invalid argument `'-Dkey=value'`. Arguments must be prefixed with either - or --.

How its possible to pass system property?

**Comment by ryanemerson** [comment, marked as answer]
Hi @amarkevich. This is a bug, I have created [ISPN-15113](https://issues.redhat.com/browse/ISPN-15113) to track this.

As a workaround you can specify your properties via a file:

```
  -P, --properties=<file>       Sets system properties from the specified file.

```
